backend/handlers/handlers.go

```diff
@@ -197,6 +197,7 @@ func NewHandlers(
 	getAPIRouter(apiRouter)("/coins/tbtc/headers/status", handlers.getHeadersStatus(coinpkg.CodeTBTC)).Methods("GET")
 	getAPIRouter(apiRouter)("/coins/ltc/headers/status", handlers.getHeadersStatus(coinpkg.CodeLTC)).Methods("GET")
 	getAPIRouter(apiRouter)("/coins/btc/headers/status", handlers.getHeadersStatus(coinpkg.CodeBTC)).Methods("GET")
+	getAPIRouter(apiRouter)("/coins/btc/set-unit", handlers.getBtcFormatUnit).Methods("GET")
```

Comment:
since this makes state changes, it should be a POST request

Reply:
Right, changed